Xamarin.Forms.Platform.iOS/Renderers/EditorRenderer.cs

```diff
@@ -60,6 +62,7 @@ protected override void OnElementChanged(ElementChangedEventArgs<Editor> e)
 				Control.Changed += HandleChanged;
 				Control.Started += OnStarted;
 				Control.Ended += OnEnded;
+                Control.ShouldChangeText += ShouldChangeText;
```

Comment:
it looks like you do not do anything on property change

Reply:
Not sure what you mean here. The event is hooked up? If you mean that I don't act on the MaxLength being changed you're right. Wasn't sure what to do with that. The logical thing to do would be to cut off the current value if its length exceeds the new MaxLength value?

Reply:
> The logical thing to do would be to cut off the current value if its length exceeds the new MaxLength value
that's what I'd do

Reply:
Definitely makes sense, implemented it